components/checkbox/README.md

```diff
@@ -40,15 +40,20 @@ Based on [research](https://designnotes.blog.gov.uk/2016/11/30/weve-updated-the-
 This checkbox has been built according to [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/) standards and all government services should strive to meet this level.  This component successfully includes the following accessibility features:
 
 ### Screenreaders
-* Proper ALT labels are included
-* Language tag has been set to english
-* List items and checkbox status are labelled properly
+As read using ChromeVox
+> > "This is a selected checkbox, checkbox checked."
 
-### Colour Contrast
-* Contrast ratios are at least 7:1 for normal text and 4.5:1 for large text
+> > "This is an unselected checkbox, checkbox not checked."
+
+> > "This is an unselected checkbox, checkbox not checked."
+
+> > "This is an unselected checkbox, checkbox not checked."
 
-### Keyboard Navigation
-* Supports tab navigation for use without a mouse
+### Target Areas
+* Checkbox size is larger than default settings to provide a larger target area
+
+### Colour Contrast
+* When checked, the checkmark exceeds a [contrast ratio](https://webaim.org/articles/contrast/) of at least 7:1 with the blue checkbox fill
```

Comment:
This isn't just for checked because the unchecked order also has the same contrast ratio